src/commands/buildCommands/configs/rules.js

```diff
@@ -20,25 +20,23 @@ module.exports = {
 					minimize: true
 				}
 			},
+			postCssLoaderConfig,
+			'sass-loader'
+		]
+	},
+	scss: {
```

Comment:
Because this rule targets only `main.scss`, it might be nice to call the rule `baseScss` or something to indicate the narrow scope of what the loader handles